### ISSUE TITLE
[Feature request] tight view

### ISSUE THREAD

**Issue by edmundlaugasson** [telegramdesktop/tdesktop#1869]
### Steps to reproduce
1. chat in group with hundreds, thousands of people simultaneously
2. when there is active conversation
3. it would be relatively hard to follow with current layout scheme
### Expected behaviour

The look would work if there could be e.g. Skype-like, IRC-like etc view designs available. Then there would be much easier to read group chat where is intensive conversation among hundreds or even thousands of simultaneous participants.
### Actual behaviour

Currently there is no tight view.
### Configuration

Linux. I am using 64-bit Linux Mint 17.3 MATE and also Lubuntu 14.04 LTS but it applies to all Linux versions

I guess this applies to all platform versions but could not test all platforms.

**Version of Telegram Desktop:**
0.9.40
### Logs

Logs are not necessary for current case.

**Comment by auchri** [comment]
Do you mean something like this:

![sch](https://cloud.githubusercontent.com/assets/2931982/13196991/0724231c-d7e0-11e5-89f0-2a49b43edd29.png)

**Comment by edmundlaugasson** [comment]
Actually even more tight - without colored background of each message. Current mode should remain as "default" (someone would like to use it anyway) but there should be more compact view(s) available in order to make Telegram usable in case of larger and intensive conversations.
E.g. [Skype conversation](https://www.digitalfirst.com/wp-content/uploads/2014/08/skype-conversation.jpg), [IRC conversation](http://jessielouthan.com/wp-content/uploads/2014/02/IRC-Chat-Room.jpg), etc.
It would be useful to make fond also smaller than currently. Or even change the font to make it more readable.

**Comment by stek29** [comment]
So they don't go up in list when new messages arrive?

**Comment by stek29** [comment]
I just understood that "no bubble" mode was meant -- which would be added at some point if I remember correctly.

**Comment by yashpalgoyal1304** [comment]
why was this issue closed? due to waiting for answer?? (but that shouldn't be the case, since it was answered here: https://github.com/telegramdesktop/tdesktop/issues/1869#issuecomment-206591112)
